CMake/ITKModuleRemote.cmake

```diff
@@ -230,9 +238,6 @@ function(itk_fetch_module _name _description)
     if(DEFINED REMOTE_GIT_TAG_${_name})
       set_property(CACHE REMOTE_GIT_TAG_${_name} PROPERTY TYPE INTERNAL)
     endif()
-    if(DEFINED Module_${_name})
```

Comment:
This was the primary bug.  If the module was off, it was also marked as an internal variable.  THAT IS A MISTAKE!